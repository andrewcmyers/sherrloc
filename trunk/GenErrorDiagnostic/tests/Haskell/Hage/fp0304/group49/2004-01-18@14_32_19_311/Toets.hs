module Toets where

verwijderRec :: (a -> a -> Bool) -> a -> [a] -> [a]
verwijderRec _ _ [] = []
verwijderRec v e (x:xs) | e `v` x =  verwijderRec v e xs
                        | otherwise = x : verwijderRec v e xs

verwijderStFun :: (a -> a -> Bool) -> a -> [a] -> [a]
verwijderStFun v e xs = filter (not.(e `v`)) xs

(%%) :: Int -> Int -> Bool
infixr 8 %%
x %% y = x rem y == 0

-- should be: x `rem` y
-- 13,10-16   13,10-10
