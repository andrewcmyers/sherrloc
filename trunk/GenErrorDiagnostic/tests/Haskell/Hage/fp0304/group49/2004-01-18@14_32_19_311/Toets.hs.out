Compiling Toets.hs
(13,10)-(13,11): Type error in application
 expression       : x rem y
 term             : x
   type           : Int                              
   does not match : (Int -> Int -> Int) -> Int -> Int
 because          : it is not a function

Compilation failed with 1 error
